### PR TITLE
Add mox.tv ad network integration for amp-ad

### PR DESCRIPTION
**Pull request by llays**

### CODE REVIEW

File: ads/_config.js [hunk 1]
```diff
@@ -642,6 +642,8 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  'mox': {},
```
[the remark this answers is not in the document]
Comment (zhouyx) [reply ads/_config.js]: More information can be found at
https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis
and
https://github.com/ampproject/amphtml/blob/master/ads/README.md#preconnect-and-prefetch

[the remark this answers is not in the document]
Comment (llays) [reply ads/_config.js]: Hello!
I've added some improvements to config but integration tests has failed.
I didn't understand an error explanation and don't know how to fix it ☹️

**Comment by torch2424** [comment]
@llays 

Hello! So the integration test was a flaky test on our end, so no worries! 😄 

However, I went ahead and tested the ad manually, and got the following:

screen shot 2018-12-27 at 11 56 49 am
screen shot 2018-12-27 at 11 57 12 am

So first, I noticed the ad is only 1px tall, so we cannot see the ad, without manually making it larger in the example. Also, we noticed renderStart is never being called. I tried this in mobile device emulation, so I am not sure if it is something wrong with the server, or the AMP implementation. 🤔 Either way, let me know how we can resolve these issues, and I think it will be good to go! 👍 

Thank you!

**Comment by llays** [comment]
Hi! 
It's our test ad campaign with 100% fill rate, but it looks like our targeting filters aren't configured well.
Can you please recheck ad using VPN from other country and/or select another device in emulation instead of iPhone 5/se?
Thank you!

**Comment by torch2424** [comment]
@llays Hello!

So I tried on multiple different device emulations in the dev tools, two real life devices (Pixel 2 Android, iPhone 6 SE), and through a Candian VPN I have set up. And the ad was still not getting filled on my end unfortunately 😢 

Is everything working on your end when you serve `examples/ads.amp.html`?

**Comment by torch2424** [comment]
@llays 

Thank you for fixing that! Also, thank you for checking the example before pushing, helps us a lot 😄 

Going to merge this in, thank you very much for your contribution! 👍

**Comment by llays** [comment]
@torch2424 

Thank you very much for participating! 🙌👍

**Pull request merged**